app/javascript/onboarding/components/IntroSlide.jsx

```diff
@@ -55,3 +184,5 @@ IntroSlide.propTypes = {
 };
 
 export default IntroSlide;
+
+/* eslint-enable camelcase */
```

Comment:
Instead of disabling and enabling the eslint camelCase rule, why not just rename the state properties `checked_code_of_conduct`, `checked_terms_and_conditions`. Maybe I'm missing something.

Reply:
This code was preexisting but I believe the reason for this is because we use the attributes on the `state` to update the user, like we do [here](https://github.com/thepracticaldev/dev.to/pull/7294/files#diff-d16a96f0524905d872d0a314d8b10189R34). The properties are camel cased on the Rails side.

Reply:
I guess the thing is that we _could_ rename the properties but then we have to construct a new object when we update the user on the backend with keys that are snakecased (because that's what Rails expects), and then we are renaming the properties to camel case only to map them back to their snakecased versions again 🤷

Reply:
In that case, leave it as is. 😄